chapters/en/slides/module7/module7_03.md

```diff
@@ -0,0 +1,389 @@
+
+Here we’re adding a set of radio buttons for the maturity rating of the
+movies and a dropdown button for the genre selection.
+
+We create the radio buttons and dropdown separately, and then add then
+bind them both to the `selection_single` object to their respective
+fields.
+
+To trigger a selection, we now need to both click a radio button **and**
+make a selection in the dropdown.
+
+---
+
+## Adding sliders allows us to highlight points based on quantitative value comparisons
+
+``` python
+slider = alt.binding_range(name='Tomatometer ')
+select_rating = alt.selection_single(
+    fields=['Rotten_Tomatoes_Rating'],
+    bind=slider)
+
+points.encode(
+    opacity=alt.condition(select_rating, alt.value(0.7), alt.value(0.1))
+).add_selection(select_rating)
+```
+
+<iframe src="/module7/charts/03/unnamed-chunk-6.html" width="100%" height="420px" style="border:none;">
+</iframe>
+
+Notes: So far we have seen how to select categorical values from
+dropdown menus and radio buttons. To instead highlight based on
+quantitative values, we can use a slider.
+
+Adding a slider is similar to adding a dropdown or set of radio buttons,
+and we do it via `binding_range()`. Here we link the slider to the same
+column as on the x-axis, so that you can see how it works.
+
+If you drag the slider around, you can see that the default behavior is
+to only highlight points that are of the exact values as the slider.
+
+This is useful for selection widgets like the dropdown, but for the
+slider we want to make comparisons such as bigger and smaller than.
+
+Let’s see how to do that in the next slide.
+
+---
+
+## Highlighting points smaller or bigger than a slider value
+
+``` python
+points.encode(
+    opacity=alt.condition(
+        alt.datum.Rotten_Tomatoes_Rating < select_rating.Rotten_Tomatoes_Rating,
+        alt.value(0.7), alt.value(0.1))
+).add_selection(select_rating)
+```
+
+<iframe src="/module7/charts/03/unnamed-chunk-7.html" width="100%" height="420px" style="border:none;">
+</iframe>
+
+Notes: To highlight values that are smaller than the slider, we need to
+compare the current value of the slider with the value of the data
+points.
+
+The current value of the slider can be accessed via
+`select_rating.Rotten_Tomatoes_Rating`, and Altair has a special `datum`
+object to access values from different columns in the dataframe. To
+compare against the column with the Rotten Tomatoes Ratings, we use
+`alt.datum.Rotten_Tomatoes_Rating`.
+
+This might seem cumbersome at first, but it is necessary to avoid
+ambiguity with only writing `"Rotten_Tomatoes_Rating"`, which in
+comparison expressions means just the string and is not a reference to a
+column in the dataframe.
+
+Now that we drag the slider you can see that all the points are
+highlighted that have a value less than the slider value.
+
+---
+
+## Customizing a slider widget
+
+``` python
+slider = alt.binding_range(name='Tomatometer ', min=10, max=60, step=10)
```

Comment:
Why does the slider only go up until 60?

And again, do we want to show more code than this if we have space?